torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2285,6 +2285,24 @@ def sample_inputs_stack(op_info, device, dtype, requires_grad, **kwargs):
         for dim in range(-1, len(shape) - 1):
             yield SampleInput(tensors, args=(dim,))
 
+
+def sample_inputs_chunk_cat(op_info, device, dtype, requires_grad, **kwargs):
```

Comment:
I would add a test with negative dim.
Also other irregularity patters in the inputs like different number of dimensions